jobs/README.md

```diff
@@ -4,6 +4,23 @@ In this section you can find some job openings listed by our community members.
 
 __Be advised that Job Openings older than 3 months will be removed for the sake of keeping current. Feel free to repost.__
 
+## QA Engineer (Heraklion or Athens) @ [Sunlight](https://www.sunlight.io)
+
+Sunlight.io is looking for a talented QA engineer to work in a totally chalenging enviroment, preferably in Heraklion, but Athens works for us as well.
+
+[Read more & Apply here](mailto:cv@sunlight.io)
+
+* PubDate: 17/12/2019
+
+## Support Engineer (Heraklion or Athens) @ [Sunlight](https://www.sunlight.io)
+
+Sunlight.io is looking for a naturaly inquirying Support engineer to work closesly with our clients, preferably in Athens, but Heraklion works for us as well.
+
+[Read more & Apply here](mailto:cv@sunlight.io)
```

Comment:
@sonaht Not very important, but is this `mailto` link correct? You already have a [link to Workable](https://apply.workable.com/sunlight-io/j/8B8E890618/) on your website, so it might be better for the applicants to be directed to that one.

Reply:
@gvre morning, by mistake included ads as well. I have removed them from the original merge. I will do a completely separate one to keep these straight. Thanks for picking it up !!!